ui/appui-abstract/src/appui-abstract/backstage/BackstageItemsManager.ts

```diff
@@ -29,6 +31,8 @@ const isInstance = <T>(args: InstanceOrArray<T>): args is T => {
 /**
  * Controls backstage items.
  * @internal
+ * @deprecated in 4.10.x. Use `BackstageItemsManager` from `@itwin/appui-react`.
```

Comment:
Let's not recommend using internal APIs.

```suggestion
 * @deprecated in 4.10.x. Use `UiFramework.backstage` from `@itwin/appui-react`.
```